justfile

```diff
@@ -20,13 +20,14 @@ default: run
 @download-test-data:
     export AZURE_STORAGE_SAS_TOKEN="sp=rl&st=2023-11-14T18:39:07Z&se=2024-11-15T02:39:07Z&spr=https&sv=2022-11-02&sr=c&sig=0KVD7ORBM7Mx1%2BhVrVbqYcQycshhvT2XvdmrWVetiQM%3D"; \
     dvc pull data/root/ETSIPETscanner_mIEC_1.root
+    dvc pull data/root/scanner_geometry.json
```

Comment:
Can we write
`dvc pull data/root/ETSIPETscanner_mIEC_1.root data/root/scanner_geometry.json`?